### ISSUE TITLE
Model Overrides Cannot Be Applied To Parent Quads

### ISSUE THREAD

**Issue by ChloeDawn** [Chisel-Team/ConnectedTexturesMod#79]
When declaring model overrides in a model that inherits quads from a parent, the overrides are not applied, as CTM does not process the parent quads and their tint indices during the stage that the overrides are applied to the model. An example model for producing this bug can be seen below.

**Mod version:** [MC1.12.2-0.3.1.16](https://minecraft.curseforge.com/projects/ctm/files/2578680)
**Forge version:** [1.12.2-14.23.4.2705](http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.12.2-14.23.4.2705/forge-1.12.2-14.23.4.2705-changelog.txt)

```json
{
  "parent": "minecraft:block/grass_normal",
  "ctm_version": 1,
  "ctm_overrides": {
    "0": {
      "type": "normal",
      "extra": {
        "light": 15
      }
    }
  }
}

```

**Comment by tterrag1098** [comment]
This may have been fixed by https://github.com/Chisel-Team/ConnectedTexturesMod/commit/b2cb2fdab5de850d5edfb6a394cb755c535f4b94

Worth a test.

**Comment by tterrag1098** [comment]
Not quite, it will only work if the parent is also a CTM model, not with vanilla models. However that's likely as good as it's going to get within 1.12.

**Comment by tterrag1098** [comment]
Upon further review, no this doesn't fix the problem at all. This was to fix an issue when combining texture and model metadata, not with model inheritance. Ah well.

**Comment by DoomRater** [comment]
Issue #144 is related if not a duplicate.
